src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,635 @@
+---
+title: Install OpenTelemetry Collector on Kubernetes
+tags:
+    - Kubernetes integration
+    - New Relic K8S OpenTelemetry integration
+    - New Relic OpenTelemetry Collector
+    - NRDOT
+    - New Relic OpenTelemetry
+    - OpenTelemetry Collector
+    - OpenTelemetry Kubernetes
+    - OpenTelemetry Helm chart
+    - New Relic OpenTelemetry installation
+metaDescription: "Instrument your Kubernetes Cluster in New Relic using OpenTelemetry Collectors"
+freshnessValidatedDate: 2024-07-23
+---
+
+You can monitor your Kubernetes cluster using OpenTelemetry, which provides a standardized way to collect and send telemetry data to New Relic. This integration allows you to gain insights into your Kubernetes environment, including metrics, events, and logs. 
+
+New Relic recommends using the New Relic distribution of OpenTelemetry (NRDOT) for Kubernetes, which is pre-configured to work seamlessly with New Relic. This distribution includes all necessary components and configurations to monitor your Kubernetes cluster effectively. However, if you prefer to use your own OpenTelemetry Collector, you can do so by ensuring it includes the required components for Kubernetes monitoring.
```

Comment:
Should we add a link to NRDOT distro for K8s?
https://github.com/newrelic/nrdot-collector-releases/tree/main/distributions/nrdot-collector-k8s